ospfd/ospf_opaque.c

```diff
@@ -560,6 +560,9 @@ register_opaque_info_per_type (struct ospf_opaque_functab *functab,
       listnode_add (new->area->opaque_lsa_self, oipt);
       break;
     case OSPF_OPAQUE_AS_LSA:
+      if (IS_DEBUG_OSPF_TRACE)
+        zlog_debug ("%s: default ospf lookup. vrf_id %u",
+              __PRETTY_FUNCTION__, new->oi->ospf->vrf_id);
```

Comment:
Let's print the vrf name not the vrf id.  People think about the vrf names not the id.